CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing Guide
+
+Welcome! We are glad that you want to contribute to our project! 💖
+
+## Ways to Contribute
+
+We welcome many different types of contributions including:
+
+* Metrics definitions
+* Bug fixes
+* Documentation and Knowledge Base articles
+* Issue Triage
+* Answering questions on Slack/Mailing List
+* Web design
+* New software features
+* Communications / Social Media / Blog Posts
+* Conference coordination and support 
+
+Not everything happens through a GitHub pull request. Please read on to learn more about coming to our meetings or reaching out to us in Slack, and let's discuss how we can work together.
+
+## Getting Started
+
+CHAOSS is an unusual open source project, in that we have a lot of sub-projects and initiatives. We are much more than code. As such, we highly encourage newcomers to the project to go through our [Quickstart Guide for New Contributors](https://chaoss.community/kb-getting-started/). This will help you get a better understanding of all the moving parts here. 
+
+## Join the Conversation
+
+If you have questions, need help, or just want to talk about the project, [Slack](https://join.slack.com/t/chaoss-workspace/shared_invite/zt-r65szij9-QajX59hkZUct82b0uACA6g) is our main communication channel for the project.
+
+Another good way to join the discussion is by filing or commenting on GitHub issues or pull requests in any relevant CHAOSS repository.
+
+## Come to Meetings
+
+Absolutely everyone is welcome to come to any of our meetings. You never need an invite to join us. In fact, we want you to join us, even if you don’t have anything you feel like you want to contribute. Just being there is enough!
+
+You can find details about our meetings on the community [calendar](https://chaoss.community/chaoss-calendar/). You don’t have to turn on your video. Over time, we hope that you feel comfortable voicing your opinions, giving feedback on others’ ideas, and even sharing your own ideas, and experiences.
+
+## Request a Tour Guide
+
+CHAOSS has a program called our Tour Guides where a newcomer can request more 1:1 guidance for getting started and connecting with the group that is of interest to them. Simply reach out to our Tour Guide Coordinator, Maryblessing Okolie, on Slack and she will start the ball rolling.
```

Comment:
We should avoid having individual people mentioned in docs like this one, since people move in and out of roles and it avoids updates later. Can they reach out in the #newcomers slack channel to ask for a tour guide, instead?

Reply:
Yes, I think that makes sense! Thanks @geekygirldawn !